app/controllers/v2/application_controller.rb

```diff
@@ -5,6 +5,5 @@ module V2
   class ApplicationController < ActionController::API
     # Accept and generate JSON API data
     include ActionController::MimeResponds
-    before_action :current_user
```

Comment:
Yeah, but it does not do anything with it if it succeeds and does not do anything if it fails. Before, we used `authenticate_user!` in this spot, which raises an error if a user was not logged in, which we didn't want. Not sure why we didn't just remove the line then.